### DISCUSSION TITLE
Single dump file

### DISCUSSION THREAD

**Discussion by ciolfi** [planetscale/discussion#283] [Q&A] [answered]
Thanks for an outstanding product! With the PlanetScale CLI, is it possible to create a dump that results in a single file, i.e., a file that combines data and schema for all tables, i.e., mysqldump?

**Comment by fooforge** [comment, marked as answer]
@ciolfi Thanks for the feedback!

`pscale database dump` generates separate files to decouple your schema from the data inserts, so no, not out of the box. Though, nothing holds you back from concatenating these files if that's what you prefer: `cat file1 file2 file3 > file`.

**Reply by ciolfi** [reply]
Thanks @fooforge (Mike). Your concatenation approach was very helpful. I'm sorry that I did not mention I was working in PowerShell, which requires a slightly different syntax as shown below. Thanks to you and the post mentioned below, I've been able to create a single backup file. I can also sequence the individual PlanetScale file content so that table schema are created prior to populating each table. Thanks, again!

Thanks also to [Smi](https://stackoverflow.com/users/1128737/smi) for [SO post 218 here](https://stackoverflow.com/questions/8749929/how-do-i-concatenate-two-text-files-in-powershell): 

The following works with any plain text file (SQL, TXT, HTML, etc.):

With wildcards:
Get-Content *.sql | Set-Content combinedfile.sql

To sequence the individual PlanetScale backup files:
Get-Content c.sql, a.sql, b.sql | Set-Content combinedsql.sql

**Reply by fooforge** [reply]
@ciolfi Great! And, thank you for sharing.
